File: OpenGL ES2.0/OpenGL ES2.0应用开发实践指南/OpenGL ES2.0应用实践iOS/OpenGL ES2.0应用实践iOS/ViewController.m
//
//  ViewController.m
//  OpenGL ES2.0应用实践iOS
//
//  Created by 冯才凡 on 2019/11/26.
//  Copyright © 2019 冯才凡. All rights reserved.
//

#import "ViewController.h"
#import "OpenGLES_VC.h"
#import "OpenGL_ES_VC02.h"
#import "OpenGL_ES_VC03.h"
#import "OpenGL_ES_VC04.h"
#import "OpenGL_ES_VC05.h"
#import "OpenGL_ES_VC06.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
//    self.view.
    // Do any additional setup after loading the view.
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    [super touchesBegan:touches withEvent:event];
    
    OpenGL_ES_VC06 * vc = [OpenGL_ES_VC06 new];
    
    [self presentViewController:vc animated:true completion:nil];
}

@end

File: Avplayer/Avplay/Avplay/ViewController.swift
//
//  ViewController.swift
//  Avplay
//
//  Created by 冯才凡 on 2019/6/6.
//  Copyright © 2019 冯才凡. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        
        let playview = FCFAVPlayerView(frame: CGRect(0,200,view.frame.size.width,200,view.frame.size.width/2.0))
//        playview.url = 
        self.view.addSubview(playview)
        
    }


}

